rp_recruit/apps.py

```diff
@@ -2,4 +2,4 @@
 
 
 class RpRecruitConfig(AppConfig):
-    name = "Recruit"
+    name = "rp_recruit"
```

Comment:
Is this change necessary?

Reply:
Yes it is, as i've upgraded django version and Recruit is not an app so it throws an error:

django.core.exceptions.ImproperlyConfigured: Cannot import 'Recruit'. Check that 'rp_recruit.apps.RpRecruitConfig.name' is correct.